pkg/granted/tokens.go

```diff
@@ -54,7 +55,7 @@ var ListSSOTokensCommand = cli.Command{
 		}
 
 		for _, key := range keys {
-			fmt.Fprintf(os.Stderr, "%s\n", fmt.Sprintf("%-*s (%s)", max, key, strings.Join(startUrlMap[key], ", ")))
+			clio.Logf("%-*s (%s)", max, key, strings.Join(startUrlMap[key], ", "))
```

Comment:
@chrnorm After updating to the new clio version, I realised that clio.Log still defaults to os.stdout, in this usecase I just want to be able to log to stdErr without a prefix or colour.

I propose that clio.Log should just default to stderr like all the other commands, and we should have some other way to log to stdout. 

What do you think?